packages/cms-cli/commands/init.js

```diff
@@ -0,0 +1,69 @@
+const { version } = require('../package.json');
+const inquirer = require('inquirer');
+const {
+  getConfigPath,
+  writeNewPortalApiKeyConfig,
+} = require('@hubspot/cms-lib/lib/config');
+const {
+  logFileSystemErrorInstance,
+} = require('@hubspot/cms-lib/errorHandlers');
+const {
+  trackCommandUsage,
+  addHelpUsageTracking,
+} = require('../lib/usageTracking');
+const {
+  DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME,
+} = require('@hubspot/cms-lib/lib/constants');
+const { PORTAL_API_KEY, PORTAL_ID, PORTAL_NAME } = require('../lib/prompts');
+const { addLoggerOptions, setLogLevel } = require('../lib/commonOpts');
+const { logDebugInfo } = require('../lib/debugInfo');
+const { logger } = require('@hubspot/cms-lib/logger');
```

Comment:
nit: let's order the imports so that it is:

1. external npm packages
2. internal npm packages (e.g. `@hubspot/cms-lib/*`)
3. local